draft-ietf-tls-tls13.md

```diff
@@ -4128,9 +4212,174 @@ Section 7 of [RFC6066], as it is not applicable to AEAD ciphers and has
+these properties are not necessarily independent, but reflect
+the protocol consumers' needs.
+
+Establishing the same session key.
+: The handshake needs to output the same session key on both sides of the
+handshake (See {{CK01}}; defn 1, part 1).
+
+Secrecy of the session key.
+: The shared session key should be known only to the communicating
+parties, not to the attacker (See {{CK01}}; defn 1, part 2).  Note that
+in a unilaterally authenticated connection, the attacker can establish
+its own session keys with the server, but those session keys are
+distinct from those established by the client.
+
+Peer Authentication.
+: The client's view of the peer identity should reflect the server's
+identity. If the client is authenticated, the server's view of the
+peer identity should match the client's identity.
+
+Uniqueness of the session key:
+: Any two distinct handshakes should produce distinct, unrelated session
+keys [REF: Need cite.]
+
+Downgrade protection.
+: The cryptographic parameters should be the same on both sides and
+should be the same as if the peers had been communicating in the
+absence of an attack (See {{BBFKZG16}}; defns 8 and 9}).
+
+Forward secret
+: If the long-term authentication keys (signature keys in certificate-based
+authentication modes or the PSK in PSK-(EC)DHE modes) are compromised after
+the handshake is complete, this does not compromise the security of the
+session key (See {{DOW92}}).
+
+Protection of endpoint identities.
+: The server's identity (certificate) should be protected against passive
+attackers. The client's identity should be protected against both passive
+and active attackers.
+{:br}
+
+Informally, the signature-based modes of TLS 1.3 provide for the
+establishment of a unique, secret, shared, key established by an
+(EC)DHE key exchange and authenticated by the server's signature over
+the handshake transcript, as well as tied to the server's identity by
+a MAC. If the client is authenticated by a certificate, it also signs
+over the handshake transcript and provides a MAC tied to both
+identities. {{SIGMA}} describes the analysis of this type of key
+exchange protocol. If fresh (EC)DHE keys are used for each connection,
+then the output keys are forward secret.
+
+The PSK and resumption-PSK modes bootstrap from a long-term shared
+secret into a unique per-connection short-term session key. This
+secret may have been established in a previous handshake. If
+PSK-(EC)DHE modes are used, this session key will also be forward
+secret. The resumption-PSK mode has been designed so that the
+resumption master secret computed by connection N and needed to form
+connection N+1 is separate from the traffic keys used by connection N,
+thus providing forward secrecy between the connections.
+
+For all handshake modes, the Finished MAC (and where present, the
+signature), prevents downgrade attacks. In addition, the random
+tainting mechanism described in {{server-hello}} allows the detection
```

Comment:
"Random tainting" sounds like you're tainting at random :)  How about: "The use of certain bytes in the client and server random to indicate TLS 1.3 support ..."